lib/services/elastic.js

```diff
@@ -5,7 +5,7 @@ const elastic = require('elasticsearch'),
   bluebird = require('bluebird'),
   _ = require('lodash'),
   util = require('util'),
-  log = require('./log').withStandardPrefix(__dirname),
+  log = require('./log').setup({file: __filename}),
```

Comment:
all of these are calling `.setup()` with `file: __filename`, but I don't see that in the clay-log docs. Are there other options for setting up logs, and do they need to be "set up" each time they're required? I feel like the api could be simplified, if so...

Reply:
Setup is simply a function exported by the log service inside of Amphora Search. What it does is allow additional metadata to be injected into the log instance for ONLY that file. It's the exact same as the old `withStandardPrefix`, but instead of a single string you could send across an object with any pertinent information. The `setup` function then uses the Clay Log api to append that.